assembly/src/ast/instruction/print.rs

```diff
@@ -290,7 +290,7 @@ impl PrettyPrint for Instruction {
                 const_text("exec") + const_text(".") + text(format!("{}::{}", module, name))
             }
             Self::Exec(InvocationTarget::AbsoluteProcedurePath { name, path }) => {
-                const_text("exec") + const_text(".") + text(format!("{}::{}", path.last(), name))
+                const_text("exec") + const_text(".") + text(format!("::{}::{}", path.last(), name))
```

Comment:
Shouldn't this output the full path rather than just the last leg? (this could also apply to other changes in this file).

Reply:
Good catch - prior to the assembler refactor it was expected that an import would have aliased the last component, but that is no longer the case.